publication-commons/src/main/java/no/unit/nva/publication/ServiceEnvironmentConstants.java

```diff
@@ -1,34 +1,32 @@
-package no.unit.nva.publication.storage.model;
+package no.unit.nva.publication;
 
 import nva.commons.core.Environment;
 import nva.commons.core.JacocoGenerated;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public final class StorageModelConstants {
+public final class ServiceEnvironmentConstants {
```

Comment:
Class moved and renamed because the URI ID is constructed after the object is read from the database.